[DOCS-1620] Improve accessibility of images (#1429)
wandb/docs: content/en/guides/core/registry/link_version.md

  {{% /tab %}}
  {{% tab header="Registry App" %}}
1. Navigate to the Registry App.

>     {{< img src="/images/registry/navigate_to_registry_app.png" alt="Navigation to Registry App showing access path from main workspace interface" >}}

Author: ```suggestion
    {{< img src="/images/registry/navigate_to_registry_app.png" alt="Demonstration of navigating to Registry" >}}
```
